spec/fluent/plugin/prometheus_metrics_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+require 'fluent/plugin/in_prometheus'
+require 'fluent/test/driver/input'
+
+require 'net/http'
+
+describe Fluent::Plugin::PromMetricsAggregator do
+
+  metrics_worker_1 = %[# TYPE fluentd_status_buffer_queue_length gauge
+# HELP fluentd_status_buffer_queue_length Current buffer queue length.
+fluentd_status_buffer_queue_length{host="0123456789ab",worker_id="0",plugin_id="plugin-1",plugin_category="output",type="s3"} 0.0
```

Comment:
probably you assume that users set explicitly `worker_id` for each metrics in the configurations.
How about adding them by default for multi workers mode?

Reply:
Good catch! I never used custom metrics from prometheus output/filter plugin so I totally missed this. I'll add worker_id to all metrics in multi worker mode and will complete the tests.

Reply:
Would you prefer to have worker_id added by default only in this new /aggregated_metrics endpoint or also to all /metrics endpoints (for each worker)? It seems cleaner to me to have it added everywhere but that would change the exported metrics for existing setups with multi workers enabled.

Reply:
yes, please add worker_id to both.

Reply:
Added worker_id when multi workers is enabled in a22766832f96a9278b571bc8c0070d8a0087765d